changelogs/fragments/6049-dconf-strings.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

This is now definitely a feature:
```suggestion
minor_changes:
```